packages/components/modal/src/drawer.tsx

```diff
@@ -49,7 +50,9 @@ export type DrawerProps = Omit<
   "scrollBehavior" | "animation" | "outside" | keyof ThemeProps
 > &
   ThemeProps<"Drawer"> &
-  DrawerOptions
+  DrawerOptions & {
+    closeOnDrag?: boolean
+  }
```

Comment:
```suggestion
  "scrollBehavior" | "animation" | "outside" | keyof ThemeProps
> &
  ThemeProps<"Drawer"> &
  DrawerOptions
```

`closeOnDrag?: boolean` goes to type DrawerOptions. Please also add JSDoc and default value.